addons/etcd/dataprotection/backup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -ex
+
+cat /etc/datasafed/datasafed.conf
+toolConfig=/etc/datasafed/datasafed.conf
```

Comment:
where is datasafed.conf?
Are these two lines of code useful?

Reply:
Oh, it's only for test, removed.